skimage/morphology/convex_hull.py

```diff
@@ -88,6 +107,11 @@ def convex_hull_image(image, offset_coordinates=True, tolerance=1e-10):
         Tolerance when determining whether a point is inside the hull. Due
         to numerical floating point errors, a tolerance of 0 can result in
         some points erroneously being classified as being outside the hull.
+    fast_drawing : bool or None, optional
+        If ``True``, use the fast SciPy function ``polygon_perimeter`` to locate
```

Comment:
not sure I understand here, `polygon_perimeter` is a scikit-image function, not a scipy one? I had the impression that the fast scipy function was `binary_fill_holes`.